src/vscode-bicep/package.json

```diff
@@ -72,11 +72,16 @@
       "type": "object",
       "title": "Bicep",
       "properties": {
-        "bicep.decompileOnPaste": {
+        "bicep.experimental.decompileOnPaste": {
           "type": "boolean",
-          "description": "Automatically convert pasted JSON Azure templates or resources into Bicep (use Undo to revert)",
+          "description": "(Experimental) NOTE: 'Enable Paste as Bicep' must also be enabled. Automatically convert pasted JSON Azure templates or resources into Bicep (use Undo to revert)",
           "default": true
         },
+        "bicep.experimental.enablePasteAsBicep": {
+          "type": "boolean",
+          "description": "(Experimental) Enable Paste as Bicep",
```

Comment:
Can we update "(Experimental) Enable Paste as Bicep" => "(Experimental) Enable Paste JSON as Bicep" to match the command title?
![image](https://user-images.githubusercontent.com/30270536/214947522-2efc0f39-543c-439d-aa9b-bd11fa5d1787.png)

Reply:
good point